website/content/docs/lambda/invoke-from-lambda.mdx

```diff
@@ -0,0 +1,271 @@
+    ```
+
+    </CodeBlockConfig>
+
+1. Issue the `terraform apply` command and Consul automatically configures a service for the Lambda function.
+
+
+### Lambda extension configuration
+
+Define the following environment variables in your Lambda functions to configure the Lambda extension. The variables apply to each Lambda function in your environment:
+
+| Variable | Description | Default |
+| ---      | ---         | ---     |
+| `CONSUL_MESH_GATEWAY_URI` | Specifies the URI where the mesh gateways that the plugin makes requests are running. The mesh gateway should be registered in the same Consul datacenter and partition that the service is running in. For optimal performance, this mesh gateway should run in the same AWS region. | none |
+| <nobr>`CONSUL_EXTENSION_DATA_PREFIX`</nobr> | Specifies the prefix that the plugin pulls configuration data from. The data must be located in the following directory: <br/>`“${CONSUL_EXTENSION_DATA_PREFIX}/${CONSUL_SERVICE_PARTITION}/${CONSUL_SERVICE_NAMESPACE}/<lambda-function-name>”` | none |
+| `CONSUL_SERVICE_NAMESPACE` | Specifies the Consul namespace the service is registered into. | `default` |
+| `CONSUL_SERVICE_PARTITION` | Specifies the Consul partition the service is registered into. | `default` |
+| `CONSUL_REFRESH_FREQUENCY` | Specifies the amount of time the extension waits before re-pulling data from the Parameter Store. Use [Go `time.Duration`](https://pkg.go.dev/time@go1.19.1#ParseDuration) string values, for example, `”30s”`. <br/>The time is added to the duration configured in the Lambda registrator `sync_frequency_in_minutes` configuration. Refer to [Lambda registrator configuration options](/docs/lambda/registration/automate#lambda-registrator-configuration-options). The combined configurations determine how stale the data may become. Lambda functions can run for up to 14 hours, so we recommend configuring a value that results in acceptable staleness for certificates. | `“5m”` |
+| `CONSUL_SERVICE_UPSTREAMS` | Specifies the upstream services that the Lambda function can call. Specify the value as an unlabelled annotation according to the [`consul.hashicorp.com/connect-service-upstreams` annotation format](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) in Consul on Kubernetes. For example, `"[service-name]:[port]:[optional-datacenter]"` | none |
+
+## Invoke the Lambda function
+
+If _intentions_ are enabled in the Consul service mesh, you must create an intention that allows the Lambda function's Consul service to invoke all upstream services prior to invoking the Lambda function. Refer to [Service Mesh Intentions](/docs/connect/intentions) for additional information.
+
+There are several ways to invoke Lambda functions. In the following example, the `aws lambda invoke` CLI command invokes the function.:
+
+```shell-session
+$ aws lambda invoke --function-name lambda-registrator-2345 /dev/stdout | cat
```

Comment:
We don't want to tell users to invoke `lambda-registrator`. We should change this to the `lambda` function that was created in the terraform example above.

```suggestion
$ aws lambda invoke --function-name lambda /dev/stdout | cat
```